.teamcity/settings.kts

```diff
@@ -1088,3 +1062,100 @@ object WpDesktop_DesktopE2ETests : BuildType({
 	}
 })
 
+object WPComPlugins : Project({
+    name = "WPCom Plugins"
+    buildType(WPComPlugins_EditorToolKit)
+    params {
+        text("docker_image_wpcom", "registry.a8c.com/calypso/ci-wpcom:latest", label = "Docker image", description = "Docker image to use for the run", allowEmpty = true)
+    }
+})
+
+object WPComPlugins_EditorToolKit : BuildType({
+    name = "Editor ToolKit"
+
+    artifactRules = "editing-toolkit.zip"
+
+    vcs {
+        root(WpCalypso)
+        cleanCheckout = true
+    }
+
+    steps {
+        script {
+            name = "Prepare environment"
+            scriptContent = """
+                #!/bin/bash
+
+                # Update node
+                . "${'$'}NVM_DIR/nvm.sh" --no-use
+                nvm install
+
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+
+                # Update composer
+                composer install
```

Comment:
Do we need `composer install` both here and in `docker_image_wpcom`?

Reply:
I'm not sure because I don't know how `composer install` actually works. Assuming it works like `yarn`, the idea is that running it in the docker image will download the libraries and populate local caches, therefore running it again on checkout should be faster because stuff is cached.

But I haven't tested it enough to know it helps for sure.